packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -342,8 +348,11 @@ export const createDefaultCallingHandlers = memoizeOne(
       await disposeAllLocalPreviewViews(callClient);
     };
 
-    const onRemoveParticipant = async (userId: string): Promise<void> => {
-      await call?.removeParticipant(fromFlatCommunicationIdentifier(userId));
+    const onRemoveParticipant = async (userId: string | CommunicationIdentifier): Promise<void> => {
+      if (typeof userId === 'string') {
+        userId = fromFlatCommunicationIdentifier(userId);
```

Comment:
instead of mutating the argument (which will change it for the caller of this function also), instead store in a local variable

Reply:
(+in a few other places)

Reply:
Actually because it's only augmented when it is a string (which is a primitive type in javascript) it *won't* change the value for the caller... but best practice to not override the argument anyways

Reply:
Created a common function in acs-ui-common and used it everywhere. LMKWYT